README.md

```diff
@@ -1,34 +1,74 @@
-BACKEND PORTFOLIO PROJECT 
+# BACKEND PORTFOLIO PROJECT
 
-Welcome to my very first app. I have created it as part of the Northcoders bootcamp. 
+<p>Welcome to my very first app. I have created it as part of the Northcoders bootcamp.</p>
```

Comment:
don't want to talk in the first person here. Just talk about what the app is and what it does. This app was created as part of Northcoders bootcamp. It has several endpoints etc etc